whisper_server.py

```diff
@@ -24,7 +24,8 @@
 ###############################################################################
 HOST = '127.0.0.1'
 PORT = 65432
-
+CLIENT_HOST = config.get_voiceattack_host()

```

Comment:
`config` object won't be available here so just move to the function where they're used. Maybe create another config item as the ip address may be different and naming won't be confusing.